pypeit/specobj.py

```diff
@@ -467,29 +467,6 @@ def apply_flux_calib(self, wave_sens, sensfunc, exptime, telluric=None, extinct_
             self[attr+'_FLAM_SIG'] = flam_sig
             self[attr+'_FLAM_IVAR'] = flam_var
 
-
```

Comment:
I'd prefer we keep this around.  We may find better performance for some instruments on an object-by-object flexure correction.